src/Controllers/CallLogsController.php

```diff
@@ -0,0 +1,366 @@
+<?php
+/*
+ * Pipedrive
+ *
+ */
+
+namespace Pipedrive\Controllers;
+
+use Pipedrive\APIException;
+use Pipedrive\APIHelper;
+use Pipedrive\Configuration;
+use Pipedrive\Models;
+use Pipedrive\Exceptions;
+use Pipedrive\Http\HttpRequest;
+use Pipedrive\Http\HttpResponse;
+use Pipedrive\Http\HttpMethod;
+use Pipedrive\Http\HttpContext;
+use Pipedrive\OAuthManager;
+use Pipedrive\Servers;
+use Pipedrive\Utils\CamelCaseHelper;
+use Unirest\Request;
+
+/**
+ * @todo Add a general description for this controller.
+ */
+class CallLogsController extends BaseController
+{
+    /**
+     * @var CallLogsController The reference to *Singleton* instance of this class
+     */
+    private static $instance;
+
+    /**
+     * Returns the *Singleton* instance of this class.
+     * @return CallLogsController The *Singleton* instance.
+     */
+    public static function getInstance()
+    {
+        if (null === static::$instance) {
+            static::$instance = new static();
+        }
+
+        return static::$instance;
+    }
+
+    /**
+     * Returns all call logs assigned to a particular user
+     *
+     * @param  array  $options    Array with all options for search
+     * @param integer $options['limit']                 (optional) For pagination, the limit of entries to be returned
+     * @param integer $options['start']                 (optional) For pagination, the position that represents the first result for the page
+     * @return void response from the API call
+     * @throws APIException Thrown if API call fails
+     */
+    public function getAllCallLogsAssignedToAParticularUser(
+        $options = [
+            'start' => 1
```

Comment:
This is a bug, default `start` should be 0. See https://pipedrive.slack.com/archives/C01C1UEEGQ1/p1625214040085000?thread_ts=1625211088.083500&cid=C01C1UEEGQ1

> GET /v1/callLogs has an optional start parameter. But it does not allow 0 ("error": "\"start\" must be a positive number"):smile:
> That means that if I have 3 call logs and try to get them using start param, I can’t - since I have to use 1 - I get only 2 records